### PR TITLE
Remove campaigns => changeset mapping

### PR DESCRIPTION
**Pull request by eseliger**
This is a duplicative source of truth to the IDs stored in the campaign and can cause inconsistencies that seem hard to debug. Also, selfishly it gives me the benefit that I can just upsert all changesets in the ApplyCampaign method, also to-be-closed/detached ones, because there is no need to distinguish them from attached ones anymore.

..Maybe this is step 0 towards a proper join table?

### CODE REVIEW

**Comment by eseliger** [comment]
Help I stalled the test suite and don't know why :|

File: migrations/frontend/1528395757_drop_campaigns_changeset_ids.up.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+DROP TRIGGER IF EXISTS trig_delete_campaign_reference_on_changesets ON campaigns;
+DROP FUNCTION IF EXISTS delete_campaign_reference_on_changesets();
```
Comment (mrnugget) [inline migrations/frontend/1528395757_drop_campaigns_changeset_ids.up.sql:4]: Why remove this too? We still have `changesets.campaign_ids`, right?

**Review by mrnugget — APPROVED** [review]
Oh, wow! We really don't need this anymore? Nice!

File: migrations/frontend/1528395757_drop_campaigns_changeset_ids.up.sql (hunk 1 above)
Comment (eseliger) [reply migrations/frontend/1528395757_drop_campaigns_changeset_ids.up.sql:4]: Argh I misunderstood what it does. I thought it keeps those two in sync, without looking further 🤦

**Comment by LawnGnome** [comment]
Campaign-changeset relationships before this PR:

giphy

**Pull request merged**